app/src/main/res/menu/main.xml

```diff
@@ -13,7 +13,7 @@
         android:id="@id/menu_refresh"
         android:checkable="false"
         android:orderInCategory="5"
-        android:title="@string/menu_refresh"
+        android:title=""
```

Comment:
taking this out? might as well rip out the whole thing
I had stuck "Create new ride" in this code but I guess that doesn't make sense.

Reply:
ya not needed